src/main/java/com/limechain/network/protocol/state/StateMessages.java

```diff
@@ -22,12 +23,12 @@ public StateMessages(String protocolId, StateProtocol protocol) {
     }
 
     public SyncMessage.StateResponse remoteStateRequest(Host us, AddressBook addrs, PeerId peer,
-                                                        String blockHash) {
+                                                        String blockHash, ByteString after) {
```

Comment:
```suggestion
                                                        String blockHash,
                                                        ByteString after) {
```
Maybe this could be one of the style guidelines of the project. If we have too many method params for a single line we separate them on 1 per line. Imo it would look better than having different number of params on each line.